.circleci/config.yml

```diff
@@ -449,6 +453,7 @@ jobs:
         # Bring up ganache
       - image: redis:3.0-alpine
       - image: trufflesuite/ganache-cli:latest
+        command: ["--port=8555", "-a", "100", "-l", "8000000"]
```

Comment:
this was to run CI with test_local configuration for ganache instead of development